src/__main__.py

```diff
@@ -135,7 +135,15 @@ def start_daemon():
 
         # setting up thread listening for updates
         global user_id
-        user_id = db.dbx.users_get_current_account().account_id
+        tmp = db.dbx.users_get_current_account().account_id.split(":")[1]
```

Comment:
If the result of account_id.split(":") does not have index 1. It will throw an index out of bound exception